resources/install.rb

```diff
@@ -4,7 +4,13 @@
 default_action :create
 
 property :yum_baseurl, String, default: 'https://jfut.integ.jp/linux/ganeti/$releasever/$basearch'
-property :yum_gpgkey, String, default: 'https://jfut.integ.jp/linux/ganeti/RPM-GPG-KEY-integ-ganeti'
+property :yum_gpgkey, String, default: lazy {
+                                         if node['platform_version'].to_i == 10
+                                           'https://raw.githubusercontent.com/jfut/ganeti-rpm/refs/heads/main/rpmbuild/integ-ganeti-release-10/SOURCES/RPM-GPG-KEY-integ-ganeti-10'
```

Comment:
They finally published this, please update it to this:
```suggestion
                                           'https://jfut.integ.jp/linux/ganeti/RPM-GPG-KEY-integ-ganeti-10'
```